src/index.html

```diff
@@ -19,42 +19,51 @@
     <form
       action="https://mate-academy-form-lesson.herokuapp.com/create-application"
       method="post"
-      class="form-wrapper first-wrapper"
+      class="form-wrapper"
       data-qa="big"
     >
-      <label for="searchOne">
+      <label
+        for="searchOne"
+        class="label-image"
+      >
         <img
-          alt="undefinied"
+          alt="search icon"
           src="images/Search.svg"
           class="search-image"
         >
       </label>
       <input
         type="text"
+        name="search-large"
```

Comment:
Generally `name` attribute should be the same as the pointing label `for` so here should be `searchOne`.